recipes/cyclus/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set version = "it" %}
+{% set sha256 = "8478396253f19735f1539cfb91e114a9982af67c9e64e3a548421a327587b77c" %}
+{% if linux %}
+    {% set libext = ".so" %}
+{% else %}
+    {% set libext = ".dylib" %}
+{% endif %}
+
+package:
+  name: cyclus
+  version: {{ version }}
+
+source:
+  fn: cyclus-{{ version }}.tar.gz
+  #url: https://github.com/cyclus/cyclus/archive/{{ version }}.tar.gz
+  url: https://github.com/scopatz/cyclus/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    #- toolchain
+    - python
+    - cmake
+    - pkg-config
+    - libxmlpp
+    - coincbc
+    - openblas
```

Comment:
Just noting this so we can come back to it later. We need to update this to match with how we deal with BLASes. Some relevant [docs]( https://conda-forge.hackpad.com/BLAS-Numpy-Friends-86De62hoNdT ). Normally the rule of thumb is mimic the numpy [recipe]( https://github.com/conda-forge/numpy-feedstock/blob/3d5a78707421c1c87ddc1eeceb8bac3cc2f59f59/recipe/meta.yaml ).